src/main/java/frc/robot/Robot.java

```diff
@@ -67,6 +70,9 @@ private void configureBindings() {
     controller.square().whileTrue(swerve.pathFindToLocation(HM_LEFT));
     controller.triangle().whileTrue(swerve.pathFindToLocation(HM_CENTER));
     controller.circle().whileTrue(swerve.pathFindToLocation(HM_RIGHT));
+
+    //Intake:
+    Intake.intake.setDefaultCommand(Intake.intake.operateIntakeCommand(()-> operator.getR2Axis(), ()-> operator.getL2Axis()));
```

Comment:
Bro what?

Create an instance at the top
Intake intake = new Intake();

And replace the Intake.intake with just intake

Reply:
Simpleton yoav stuff lol

Reply:
Were not singeltoning